src/main/resources/db/migration/V0.0.2_003__insert_default_questionnaire.sql

```diff
@@ -3599,4 +3599,4 @@ values (
             select max(id)
             from question
         )
-    );
\ No newline at end of file
+    );
```

Comment:
we don't need to change this file just for a new line at the end

Reply:
I simply added EOF character